app/jsx/components/LazyImageComp.jsx

```diff
@@ -0,0 +1,25 @@
+// ##### Lazy Image Component ##### //
+
+import React from 'react'
+import lozad from 'lozad'
+
+class LazyImageComp extends React.Component {
+  constructor() {
+    super();
+    this.observer = lozad('.c-lazyimage');
```

Comment:
This should probably happen in componentDidMount

Reply:
Also, this should only watch the element in question. Watching all elements of class "c-lazyimage" (which is what CSS selector `.c-lazyimage` does) is inefficient at best, and nasty at worst. Witness the similar problem we had with dotdotdot in this regard.

Reply:
Another thing (sorry there are so many things): Lozad should only run on the client; it gets an error when we do it on the server in iso mode.  The following if statement will evaluate to try when running in the browser: `if (!(typeof document === "undefined"))`